arangod/Replication/DatabaseInitialSyncer.cpp

```diff
@@ -500,12 +589,33 @@ Result DatabaseInitialSyncer::handleCollectionDump(arangodb::LogicalCollection*
       }
     }
     
+    // increase chunk size for next fetch
+    if (chunkSize < MaxChunkSize) {
+      chunkSize = static_cast<uint64_t>(chunkSize * 1.5);
+      if (chunkSize > MaxChunkSize) {
+        chunkSize = MaxChunkSize;
+      }
+    }
+
+    if (checkMore) {
+      ++posted;
+      SchedulerFeature::SCHEDULER->post([this, &stats, &baseUrl, &posted, dumpStatus, coll, leaderColl, batch, fromTick, chunkSize]() {
```

Comment:
Do we need to capture a shared_ptr from this?
Otherwise the Scheduler may run this task, after this has been deleted.

Reply:
this is ensured by the "posted" variable.